src/api/users/src/routes/users.js

```diff
@@ -140,6 +142,43 @@ router.put(
     const { id } = req.params;
     const { body } = req;
 
+    try {
+      const userRef = db.doc(id);
+      const doc = await userRef.get();
+
+      if (!doc.exists) {
+        next(createError(404, `user ${id} not found.`));
+      } else {
+        const user = new User(body);
+        // no user can be created as an admin by default
+        user.isAdmin = false;
+        // NOTE: doc().update() doesn't use the converter, we have to make a plain object.
+        await db.doc(id).update(user.toJSON());
+        res.status(200).json({ msg: `Updated user ${id}` });
+      }
+    } catch (err) {
+      next(err);
+    }
+  }
+);
+
+// put (update) a user with a supplied id, validated by the schema
+// rejected if a user could not be found, otherwise user updated
+// this route is only accessible by administrators
+// it allows the modification of the isAdmin property
+router.put(
+  '/:id/admin',
```

Comment:
This looks great, but one question: the naming implies that you are only setting the `admin` aspect of this record vs. all fields, yet you are allowing other fields to be updated here.  I would limit it to just the admin change.

Reply:
Also done in `updateUser`